platform/android/build.gradle.template

```diff
@@ -3,7 +3,7 @@ buildscript {
 		jcenter()
 	}
 	dependencies {
-		classpath 'com.android.tools.build:gradle:2.1.0'
+		classpath 'com.android.tools.build:gradle:2.1.2'
```

Comment:
Update classpath build tools to 2.1.2 to match the latest version of Android Studio as of this commit.

![buildtools_212](https://cloud.githubusercontent.com/assets/12244526/17388744/534386dc-5a28-11e6-91e1-8b5d69335322.png)

:-)